handlers/retest/ci.go

```diff
@@ -0,0 +1,215 @@
+package retest
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+
+	. "github.com/Huawei-PaaS/ci-bot/handlers/types"
+
+	"github.com/golang/glog"
+)
+
+const (
+	TravisCIEndPoint = "https://api.travis-ci.org" //travis-ci endpoint
+	ContentTypeJSON  = "application/json"          //content-type
+	TravisAPIVersion = "Travis-API-Version"        //API version, a Mandatory header field for Travis-CI V3 APIs call
+)
+
+type TravisJobType string
+
+//Travis Build Job Names
+const (
+	Verify      TravisJobType = "verify"   //Travis Build Job Name
+	Crossbuild  TravisJobType = "crossbuild" //Travis Build Job Name
+	Build       TravisJobType = "build"   //Travis Build Job Name
+	Unittest    TravisJobType = "unittest" //Travis Build Job Name
+	Integration TravisJobType = "integration" //Travis Build Job Name
+)
+//Travis Job Id's
+const(
+	JobBuild        = iota + 1
```

Comment:
The order of Travis Job Id is inconsistent with the name? I would suggest to keep them consistent.